opentelemetry-otlp/examples/basic-otlp/Cargo.toml

```diff
@@ -7,8 +7,10 @@ publish = false
 
 [dependencies]
 once_cell = "1.17"
-opentelemetry_api = { path = "../../../opentelemetry-api", features = ["metrics"] }
-opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio"] }
-opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics"] }
+opentelemetry_api = { path = "../../../opentelemetry-api", features = ["metrics", "logs"] }
+opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio", "logs"] }
+opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
 opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
 tokio = { version = "1.0", features = ["full"] }
+opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log"}
```

Comment:
We are not using `opentelemetry-appender-tracing` here because of instrumentation loopback issue ?

Reply:
yes. Will be opening an issue to track that.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/1171